app/docs/0.11.x/cli.md

```diff
@@ -133,11 +133,18 @@ This command prepares the Kong prefix folder, with its sub-folders and files.
 ```
 Usage: kong prepare [OPTIONS]
 
-Prepares the Kong prefix in the configured prefix directory.
+Prepare the Kong prefix in the configured prefix directory. This command can
+be used to start Kong from the nginx binary without using the 'kong start'
+command.
```

Comment:
Prepares the Kong prefix. This command can be used to set up a working directory (prefix) such that Kong can be started from the Nginx binary, as opposed to using Kong's own 'kong start' command.